Lib/test/test_enum.py

```diff
@@ -2036,6 +2036,35 @@ class GoodStrEnum(StrEnum):
             two = '2'
             three = b'3', 'ascii'
```

Comment:
I am surprised that it is not an error. Do we have a need to support this syntax?

Reply:
It's a standard Enum feature that everything after the `=` is passed to the mixed-in data type's constructor, so whatever it supports is automatically supported.  I just threw in some checks for the other `str` arguments to give friendlier error messages.